resources/qml/PrintSetupHeaderButton.qml

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.1
+import QtQuick.Controls 2.1
+
+import Cura 1.0 as Cura
+import UM 1.5 as UM
+
+ToolButton
+{
+    id: base
+
+    property alias tooltip: tooltip.text
+
+    contentItem: Label {}
```

Comment:
Uh. That label isn't doing anything. Maybe the content label should be moved out of the background?

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/b3a6d8a1fc4b032a5816d51d030646ab46bc5766
https://github.com/Ultimaker/Cura/pull/11336/commits/a66275ce73c0e0e1f8caf37c3c4f3d2a266dfe34